Wrappers/WebElement.cs

```diff
@@ -0,0 +1,130 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using System.Collections.ObjectModel;
+using System.Drawing;
+using Utilities;
+
+namespace Wrappers
+{
+    public class WebPageElement : IWebElement
+
+    {
+        protected IWebDriver Driver;
+        protected IWebElement Element;
+
+        public WebPageElement(IWebElement element)
+        {
+            Element = element;
+        }
+
+        public WebPageElement(IWebDriver driver, By locator)
+        {
+            Driver = driver;
+            Element = Driver.FindElements(locator).FirstOrDefault();
+        }
+
+        public WebPageElement()
```

Comment:
You don't need parameterless constructor. How are you suppose to use an `WebPageElement` instance having `null` in `Element` property?

Reply:
Deleted